rust/operator-binary/src/config.rs

```diff
@@ -70,9 +76,19 @@ pub fn build_bootstrap_conf(overrides: BTreeMap<String, String>) -> String {
         "java.arg.1".to_string(),
         "-Dorg.apache.jasper.compiler.disablejsr199=true".to_string(),
     );
-    // JVM memory settings
-    bootstrap.insert("java.arg.2".to_string(), "-Xms1024m".to_string());
-    bootstrap.insert("java.arg.3".to_string(), "-Xmx1024m".to_string());
+
+    // TODO: log failure to compute Java heap size
+    if let Ok(heap_size) = resource_config
+        .memory
+        .limit
+        .map(|q| to_java_heap(&q, 0.8))
+        .unwrap_or_else(|| Ok("".to_string()))
+    {
+        // JVM memory settings
+        if !heap_size.is_empty() {
+            bootstrap.insert("java.arg.3".to_string(), heap_size);
```

Comment:
1. What happened to `-Xms`?
2. Is it a problem that we jump from `java.arg.1` to either `java.arg.3` or `java.arg.4`?

Reply:
1. heap_size is formatted with -Xmx. 
2. good question. we don't jump currently since there is always a heap size.

Reply:
1. Exactly, don't we want/need to write both?
2. Doesn't that mean that we always jump from 1 to 3 now that `-Xms` is gone?

Reply:
I have a feeling we misunderstand each other here. But I hope the [latest commit](https://github.com/stackabletech/nifi-operator/pull/265/commits/cfc23541e1c05aa2e1e4ba7dbc97d801552e80a0) does what we both want :)

Reply:
That should fix concern 2, although that raises the secondary question of whether `java.arg.0` is also respected.

Reply:
What a noob mistake. It's fixed now.

Reply:
I think -Xms is still missing though?

Reply:
Yes. Do we want to merge this anyway or do we need to calculate that somehow?

Reply:
Yes, `to_java_heap` provides `-Xmx`. That doesn't really answer what happened to `-Xms`.